mtp_bank_admin/templates/q_and_a_page/q_and_a_page.html

```diff
@@ -0,0 +1,69 @@
+	<ul class="list list-bullet lede no">
+		<li>the reference you inserted for the bank transfer is wrong - check with the prisoner that you have the prisoner’s correct date of birth and correct prisoner number</li>
+		<li>the prisoner has left prison</li>
+		<li>you paid cash in-branch (not allowed)</li>
+		<li>you made an international transfer (not allowed)</li>
+	</ul>
+	<p class="lede no-space-above" >You’ll <strong>ONLY</strong> get a refund if the reference you inserted for the bank transfer is incorrect OR if the prisoner has left prison</p><br>
+
+
+	<u><h1 class="heading-medium no-space-under-heading">How long will it take for me to get a refund from my bank transfer?</h1></u>
+	<p class="lede no-space-above" >4 working days from the day you sent the money.</p><br>
+
+	<u><h1 class="heading-medium no-space-under-heading">How will I know I’ve got my refund?</h1></u>
+	<p class="lede no-space-above" >Check your bank account 4 working days after you sent the money.<br>On your bank statement you’ll see the refund appear as ‘GBS RE NOMS AGENCY’.</p><br>
+
+	<u><h1 class="heading-medium no-space-under-heading">How do I check the correct reference number for my bank transfer?</h1></u>
+	<p class="lede no-space-under-heading">Go to: <a href="https://www.gov.uk/send-prisoner-money">www.gov.uk/send-prisoner-money</a></p>
+	<p class="lede no-space-above" >Try to generate the reference.<br> If the reference can’t be generated, check with the prisoner that you have their correct date of birth and correct prisoner number.<br>Sometimes the date of birth is entered incorrectly into the prison system.<br>The prisoner might need to ask an officer on the wing to check it and let you know.</p><br>
+
+	<u><h1 class="heading-medium no-space-under-heading">How do I get my money back if I made a bank transfer in-branch or by international transfer?</h1></u>
+	<p class="lede no-space-above" >You need to talk to the prison business hub staff to identify the payment. When it’s been identified you will need to write to them confirming:</p><br>
+	<ul class="list list-bullet lede no">
+		<li>who you are</li>
+		<li>your address </li>
+		<li>how you made the payment (in-branch or international transfer)</li>
+		<li>if in-branch, what bank/branch you went to</li>
+		<li>your bank</li>
+		<li>your sort code, account number</li>
+		<li>prisoner’s name</li>
+		<li>prisoner’s prison number</li>
+	</ul>
+
+	<p class="lede no-space-above" >When the prison business hub staff get your letter, they’ll send you a cheque.  </p><br>
+
+	<u><h1 class="heading-medium no-space-under-heading">If I send money using the service how will my data be used or stored?</h1></u>
+	<p class="lede no-space-under-heading">Go to: <a href="https://www.gov.uk/send-prisoner-money">www.gov.uk/send-prisoner-money</a></p>
+	<p class="lede no-space-above" >Click on ‘Terms and Conditions’ at the bottom of the page.</p><br>
+  </div>
+
+</div>
+{% endblock %}
+  
```

Comment:
Generally you always end a file with a new line. The absence of one here is why github has put that little red sign there.